content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +942,126 @@ a scheduled action, but you can also send each invoice manually if needed.
 
 Once the invoice is validated with |SUNAT|, customers can download the :file:`.zip` with the CDR,
 XML, and PDF files directly from the customer portal by clicking the :guilabel:`Download` button.
+
+Reports
+=======
+
+Permanent Inventory Reports: PLE 12.1 and PLE 13.1
+--------------------------------------------------
+
+Two permanent inventory reports can be produced in Odoo for Peruvian accounting:
+PLE 12.1 and PLE 13.1. All inventory transactions made need to be reported. PLE 12.1 only records
+transactions by physical units, while PLE 13.1 tracks both physical units and the monetary value of
+inventory.
+
+Context
+~~~~~~~
+
+PLE 12.1: Permanent Inventory Record in Physical Units
+******************************************************
+
+This record **only tracks inventory in physical units**, focusing on the inflow and outflow of goods
+for effective management and planning. It must be maintained semi-annually (January-June and
+July-December) with monthly transaction details reported within these periods. The submission
+deadlines are October 1st for the first semester and April 1st for the second semester,
+in accordance with Superintendent Resolution 169-2015.
```

Comment:
To avoid the ambiguous "This"
```suggestion
|PLE| 12.1 **only tracks inventory in physical units**, focusing on the inflow and outflow of goods
for effective management and planning. It must be maintained semi-annually (January-June and
July-December) with monthly transaction details reported within these periods. The submission
deadlines are October 1st for the first semester and April 1st for the second semester,
in accordance with the *Superintendent Resolution 169-2015*.
```